### ISSUE TITLE
Battle script does not work

### ISSUE THREAD

**Issue by OmniStef94** [MathewSachin/Fate-Grand-Automata#485]
Before opening an issue:
- [ Y ] I tested the latest release
- [ Y ] I looked at other issues (even the closed ones)
- [ Y ] I read the [Troubleshooting Guide](https://github.com/MathewSachin/Fate-Grand-Automata/wiki/Troubleshooting)
- [ Y ] I'll fill this template properly

> Your issue will get closed/locked by the bot in case proper info isn't provided
If you open unnecessary/dumb issues, you'll be rewarded with appropriate tags.

**Describe the bug**
Issue that a friend of mine is having. Battle script does not work, whether it's in the Support screen, the actual battles or even the Battle Finish part. Only other script that could be tested was Lottery and that one worked just fine and opened up the boxes without issues, but nothing happens when activating the Battle script, no matter on what screen.

**Video**
https://www.youtube.com/watch?v=i8MajtrxPtg

Basically you'll see this from Debug, no matter the screen.

**Device Info (please fill at-least 'Device model'):**
 - Device model: Samsung Galaxy S10
 - Screen size: 1440 x 3040
 - Android Version: Android 10
 - RAM: 8GB

**Comment by reconman** [comment]
FGO and FGA are in full screen mode?

**Comment by OmniStef94** [comment]
Yup, they are.

**Comment by reconman** [comment]
Is Ignore Notch enabled? If so, disable it.

**Comment by OmniStef94** [comment]
I've tried  both enabled and disabled, issue persists.

**Comment by reconman** [comment]
Is the video with Ignore Notch or without? If it's with Ignore Notch, upload one without that option.

**Comment by OmniStef94** [comment]
It was with Ignore Notch, but just gave it a shot because we never tried without Ignore Notch after we set the whole thing to Full Screen.

Works well now. Thanks for the help!

**Comment by jdellaverson19** [comment]
How was the issue resolved? I'm running into the same issues with the same android version and whatnot.
